terraform/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "environment" {
```

Comment:
this is passed automatically and is equal to the directory name under `environments`